src/api/jokesApi.ts

```diff
@@ -4,15 +4,14 @@ import { jokesClient } from "./httpClient";
 import { PATHS } from "../constans/ApiPaths";
 
 export const jokesApi = {
-  fetchRandomJokes: async (): Promise<AxiosResponse<ChuckJoke>|AxiosError> => { 
+  fetchRandomJokes: async (params: string): Promise<AxiosResponse<ChuckJoke>|AxiosError> => { 
     try { 
-      const response = await jokesClient.get<ChuckJoke>(PATHS.RANDOM_CHUCK_JOKE_PATH);
+      const response = await jokesClient.get<ChuckJoke>(`${PATHS.RANDOM_CHUCK_JOKE_PATH}${params}`);
```

Comment:
axios client has param supports, you don't need to pass that way. Instead you can use
` await jokesClient.get<ChuckJoke>(PATHS.RANDOM_CHUCK_JOKE_PATH,{
params:{param1, param2,etc}
});`